homeassistant/components/zha/switch.py

```diff
@@ -260,8 +261,16 @@ async def async_update(self) -> None:
 class OnOffWindowDetectionFunctionConfigurationEntity(
     ZHASwitchConfigurationEntity, id_suffix="on_off_window_opened_detection"
 ):
-    """Representation of a ZHA on off transition time configuration entity."""
+    """Representation of a ZHA window detection configuration entity."""
```

Comment:
This sounds weird. Is this the "detection interval" or does it actually detects a window? Maybe "detection window"?

Reply:
I think it detects open windows for a TRV?

Reply:
@rforro what does this do? I noticed the stale doc string after I merged your PR when I was working on this PR.

Reply:
![3052C871-202B-49E9-97BD-7AF7DB0594CF](https://user-images.githubusercontent.com/1335687/170153168-e5390430-a81b-4741-ae9c-c99aec7312df.jpeg)


Yeah I think that’s what it is